modules/tinymce/src/core/main/ts/api/Tinymce.ts

```diff
@@ -99,6 +99,7 @@ export interface TinyMCE extends EditorManager {
     ControlSelection: (selection: Selection, editor: Editor) => ControlSelection;
     BookmarkManager: (selection: Selection) => BookmarkManager;
     Selection: (dom: DOMUtils, win: Window, serializer, editor: Editor) => Selection;
+    StyleSheetLoader: StyleSheetLoader;
```

Comment:
This only exposes the types, not the actual constructor. So it likely needs to be added to the actual exported JS as well (eg ~line 220).

Reply:
I'll get it right one day. 
How about now?

Reply:
Yep, looks good to me now, thanks @tinydylan! :+1:

Reply:
Cheers, mate. Thanks for you help.